shortlinks.csv

```diff
@@ -0,0 +1,46 @@
+keyword,destination_url
+shortlinks,https://github.com/hyphacoop/shortlinks/blob/master/shortlinks.csv
+keywords,https://link.hypha.coop/shortlinks
+drive,https://drive.google.com/drive/u/0/folders/14KYnYwOEK3InYZ3jCn-Gtf5q430sE9oc
+gdrive,https://link.hypha.coop/drive
+loomio,https://loomio.hypha.coop
+github,https://github.com/hyphacoop
+gh,https://github.com/hyphacoop
+handbook,https://github.com/hyphacoop/handbook
+chat,https://chat.tomesh.net/#/room/#tech-coop:tomesh.net
+template,https://raw.githubusercontent.com/hyphacoop/organizing/master/-meeting-template.md
+glossary,https://github.com/hyphacoop/handbook/issues/4
+agendas,https://github.com/hyphacoop/december-retreat
+reading-pool,https://github.com/hyphacoop/handbook/blob/master/reading-pool.md
+roles,https://github.com/hyphacoop/handbook/blob/master/roles.md
+values,https://github.com/hyphacoop/handbook/blob/master/values.md
+retreat-playbook,https://github.com/hyphacoop/organizing/blob/master/2018-december-retreat/2018-11-28-retreat-playbook.md
+linkedin,https://www.linkedin.com/company/hyphacoop/
+cron,https://github.com/hyphacoop/worker-coop-scripts
+scheduled-tasks,https://github.com/hyphacoop/worker-coop-scripts
+inventory,https://hackmd.io/@patcon/HyH0W9bAE
+leads,https://docs.google.com/spreadsheets/d/1LyElXvLj7ASclSPQqcgOnuEWqBWBStilEfOhIosZzhA/edit#gid=0
+opportunities,https://link.hypha.coop/leads
+conversions,https://docs.google.com/spreadsheets/d/1LyElXvLj7ASclSPQqcgOnuEWqBWBStilEfOhIosZzhA/edit#gid=30829554
+slogans,https://hackmd.io/s/HkRaA_woE
+chants,https://hackmd.io/s/Bk-BO6viN
+twitter,https://twitter.com/hyphacoop
+logs,https://circleci.com/gh/hyphacoop/worker-coop-scripts/tree/master
+interviews,https://docs.google.com/spreadsheets/d/1vjnT0afFh-Ss1BCYZg7nuLyjwZqzRLsSzkhej_OdoFo/edit#gid=0
+schedules,https://www.when2meet.com/?7708144-gVsJ3
+donations,https://docs.google.com/spreadsheets/d/1DpnzCpXWgQr66nFJFIlpa5XSBrt8IfZaLVZH0_q6kcA/edit#gid=2079904514
```

Comment:
How will this work once we get rid of the shortlink google sheet?

Reply:
how do you mean? Won't it just go to another sheet? Our CRM and other scratch space stuff (like donations) is still tentatively in GDocs, right?

Reply:
Oh, I understand. It's just another worksheet in the google spreadsheet. So we can remove the shortlinks sheet, and rename the overall gsheet, and the link is unaffected.

Reply:
sounds good!